spec/site_prism/page_objects/sections/cases/offender_sar/case_details_section.rb

```diff
@@ -0,0 +1,134 @@
+module PageObjects
+  module Sections
+    module Cases
+      module OffenderSAR
+        class CaseDetailsSection < SitePrism::Section
+
+        element :section_heading, '.case-details .request--heading'
+        element :original_section_heading,
+                '.original-case-details .request--heading'
+
+        section :sar_basic_details, '.sar-basic-details' do
+          section :case_type, 'tr.case-type' do
+            element :sar_trigger, 'td .sar-trigger'
+            element :data, 'td'
+          end
+
+          section :data_subject, 'tr.data-subject' do
+            element :data, 'td'
+          end
+
+          section :data_subject_type, 'tr.data-subject-type' do
+            element :data, 'td'
+          end
+
+          section :requester_name, 'tr.requester-name' do
+            element :data, 'td'
+          end
+
+          section :third_party, 'tr.third-party' do
+            element :data, 'td'
+          end
+
+          section :prison_number, 'tr.prison-number' do
+            element :data, 'td'
+          end
+
+          section :subject_aliases, 'tr.subject-aliases' do
+            element :data, 'td'
+          end
+
+          section :previous_case_numbers, 'tr.previous-case-numbers' do
+            element :data, 'td'
+          end
+
+          section :other_subject_ids, 'tr.other-subject-ids' do
+            element :data, 'td'
+          end
+
+          section :date_received, 'tr.date-received' do
+            element :data, 'td'
+          end
+
+          section :date_of_birth, 'tr.date-of-birth' do
+            element :data, 'td'
+          end
+
+          section :internal_deadline, 'tr.case-internal-deadline' do
+            element :data, 'td'
+          end
+
+          section :external_deadline, 'tr.case-external-deadline' do
+            element :data, 'td'
+          end
+
+          section :response_address, 'tr.response-address' do
+            element :data, 'td'
+          end
+
+          section :team,  'tr.team' do
+            element :data, 'td'
+          end
+        end
```

Comment:
Do these rows in the database need to be distinct sections? There's examples in the codebase of just using CSS `nth-child` style of selector, or considering every row is assigned a CSS class then just using those? Seems too granular hence inflexible.

Reply:
Again, this is how everything is already set up